src/perftest_parameters.h

```diff
@@ -439,6 +439,7 @@ struct perftest_parameters {
 	int				disable_fcs;
 	int				flows;
 	int				flows_burst;
+        int                             wait_keystroke;
```

Comment:
Could you drop commit 1db9f45f6d10faad47153c1652543ae5009f2425 and add the above ffix at line 392 , then i can merge it into petftest